core/src/main/java/org/apache/calcite/sql/dialect/HiveSqlDialect.java

```diff
@@ -80,6 +82,10 @@ public HiveSqlDialect(Context context) {
   @Override public void unparseCall(final SqlWriter writer, final SqlCall call,
       final int leftPrec, final int rightPrec) {
     switch (call.getKind()) {
+    case ARRAY_VALUE_CONSTRUCTOR:
+    case MAP_VALUE_CONSTRUCTOR:
+      unparseSparkArrayAndMap(writer, call, leftPrec, rightPrec);
```

Comment:
minor: should we think about a different name since it is not only `Spark` after this PR?

Reply:
Hi, Thanks for the review. This name means we use SparkSQL standard to handle Array and Map, Like SparkSqlDialect and StarRocksSqlDialect use unparseHiveTrim method to handle the Trim.  So I think it's Ok.